### PR TITLE
done #28 rename methods

### PR DESCRIPTION
**Pull request by ganchclub**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All Notable changes to `ElementFinder` will be documented in this file
 ## Unreleased [0.1.0-alpha.2 2016-05-26]
 ### Changed
 - #18 Skip `XpathExpression` creation. Use `CssExpression` only when needed.
+- #28 Method `ElementFinder::elements()` has been renamed to `ElementFinder::element()`, 
+- #28 Method `ElementFinder::html()` has been renamed to `ElementFinder::content()`, 
+- #28 Method `ElementFinder::query()` has been renamed to `ElementFinder::executeQuery()`,
+- #28 Method `NodeHelper::getOuterHtml()` has been renamed to `NodeHelper::getOuterContent()`,
+- #28 Method `NodeHelper::getInnerHtml()` has been renamed to `NodeHelper::getInnerContent()`,
+- #28 Added `ElementFinder::query()` as an alias of `ElementFinder::node()`
```
Comment (funivan) [inline CHANGELOG.md]: wrong section

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ All Notable changes to `ElementFinder` will be documented in this file
 ## Unreleased [0.1.0-alpha.2 2016-05-26]
 ### Changed
 - #18 Skip `XpathExpression` creation. Use `CssExpression` only when needed.
+- #28 Method `ElementFinder::elements()` has been renamed to `ElementFinder::element()`, 
+- #28 Method `ElementFinder::html()` has been renamed to `ElementFinder::content()`, 
+- #28 Method `ElementFinder::query()` has been renamed to `ElementFinder::executeQuery()`,
+- #28 Method `NodeHelper::getOuterHtml()` has been renamed to `NodeHelper::getOuterContent()`,
+- #28 Method `NodeHelper::getInnerHtml()` has been renamed to `NodeHelper::getInnerContent()`,
```
Comment (funivan) [inline CHANGELOG.md]: wrong section. Move to deprecated

File: src/ElementFinder.php [hunk 3]
```diff
@@ -300,22 +312,33 @@ public function object($xpath, $outerHtml = false) {
...
     public function node($xpath) {
-      return $this->query($xpath);
+      return self::query($xpath);
+    }
+
+
+    /**
+     * Fetch nodes from document
+     *
+     * @param $xpath
```
Comment (funivan) [inline src/ElementFinder.php]: string =)

**Comment by ganchclub** [comment]
fixed & passed

**Pull request merged**